src/test/kotlin/no/nav/klage/oppgave/repositories/KlagebehandlingRepositoryTest.kt

```diff
@@ -0,0 +1,98 @@
+package no.nav.klage.oppgave.repositories
+
+import no.nav.klage.oppgave.domain.klage.Hjemmel
+import no.nav.klage.oppgave.domain.klage.Klagebehandling
+import no.nav.klage.oppgave.domain.klage.Oppgavereferanse
+import no.nav.klage.oppgave.domain.kodeverk.Sakstype
+import no.nav.klage.oppgave.domain.oppgavekopi.OppgaveKopi
+import no.nav.klage.oppgave.domain.oppgavekopi.Prioritet
+import no.nav.klage.oppgave.domain.oppgavekopi.Status
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
+import org.springframework.test.context.ActiveProfiles
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@ActiveProfiles("local")
+@DataJpaTest
```

Comment:
Denne må nå legges til på alle DataJpaTester: `@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)`

Ellers bruker vi ikke dataSources vi har konfigurert.

Reply:
Jøss, den var ny for meg! Men jeg skjønner ikke helt, hva brukes da? Vi har jo bare H2 uansett?

Reply:
Hvis vi ikke har den, så setter Spring opp en egen H2-instans, og ikke den vi har konfigurert i application.yml i test. Da får vi ikke riktig dialekt og andre innstillinger vi forventer.